### PR TITLE
Added standard regression and Pearson correlation coefficient

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

File: src/regression_sensitivity.jl [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        params = []
+        for j in 1:length(p_range)
+            if i != j
+                push!(params,p_fixed[j])
+            else
+                push!(params,0)
+            end
+        end
+        xis = []
+        yis = Array{Float64}[]
```
Comment (ChrisRackauckas) [inline src/regression_sensitivity.jl]: this typing can be incorrect.

File: src/regression_sensitivity.jl [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        srcs = []
+        params = []
+        for j in 1:length(p_range)
+            if i != j
+                push!(params,p_fixed[j])
+            else
+                push!(params,0)
+            end
+        end
+        xis = []
```
Comment (ChrisRackauckas) [inline src/regression_sensitivity.jl]: This is untyped

File: src/regression_sensitivity.jl [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            formula *= "x$j +"
+        end
+        formula = formula[1:end-1] * ")"
+        ols_x = lm(eval(parse(formula)),df)
+        x_cap = x_lm .- predict(ols_x)
+        y_caps = []
+        for k in 1:size(yis)[1]
+            y_cps = []
+            for j in 2:size(yis)[2]
+                df_arr = Array{Float64}[Float64[] for i in 1:length(yis[k,j,:])]
```
Comment (ChrisRackauckas) [inline src/regression_sensitivity.jl]: Again many assumptions here, and this isn't strictly typed anyways.

**Comment by ChrisRackauckas** [comment]
Needs tests. Test against a linear function and make sure it gets the analytical values.

File: src/regression_sensitivity.jl (hunk 1 above)
Comment (Vaibhavdixit02) [reply src/regression_sensitivity.jl]: The typing was required for the regression model 😕

File: src/regression_sensitivity.jl (hunk 3 above)
Comment (Vaibhavdixit02) [reply src/regression_sensitivity.jl]: There were problems in the regression model creation without the typing.

**Comment by ChrisRackauckas** [comment]
I'll merge and we can continue when you're ready.

**Pull request merged**